src/scrabble.js

```diff
@@ -1,5 +1,116 @@
-function scrabble() {
+// Input word here (moved to top for ease of access)
+const scrabbleInput = 'P[i]nt'
 
+const pointsMap = {
+  A: 1,
+  E: 1,
+  I: 1,
+  O: 1,
+  U: 1,
+  L: 1,
+  N: 1,
+  R: 1,
+  S: 1,
+  T: 1,
+  D: 2,
+  G: 2,
+  B: 3,
+  C: 3,
+  M: 3,
+  P: 3,
+  F: 4,
+  H: 4,
+  V: 4,
+  W: 4,
+  Y: 4,
+  K: 5,
+  J: 8,
+  X: 8,
+  Q: 10,
+  Z: 10
+}
+
+scrabble(scrabbleInput)
+
+function scrabble(word) {
+  // Guard condition for null
+  if (word === null) {
+    console.log(`Why would you enter null? This gives you 0 points.`)
+    return 0
+  }
+
+  // If it's not null we move on
+  // Setting up initial points tally at 0
+  let points = 0
+
+  // Coverting the given word to upper case
+  const wordUpperCase = word.toUpperCase()
```

Comment:
testing.... you could chain methods here like - word.toUpperCase().split(''). Doesn't make a whole lot of difference of course, just if you want less lines

Reply:
Yeah, you're right, I may have gone overboard trying to make my thing super readable...

Reply:
For readability you're probably correct in all fairness ;) I suppose chaining together a ton of methods would be pretty unreadable

Reply:
Well, I think word.toUpperCase().split('') is probably readable even to us already and we have only been doing this for 3 weeks. Wonder what the best practice for this is, let's ask Ed